fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexScrubber.java

```diff
@@ -225,30 +233,36 @@ public Builder setEntriesScanLimit(long entriesScanLimit) {
 
             /**
              * Declare that the index to be scrubbed is valid for scrubbing, regardless of its type's name.
-             *
+             * Note: this option is associated with legacy code only (see {@link #useLegacyScrubber(boolean)}). Else the
+             * index maintainer must provide a valid {@link IndexScrubbingTools}.
              * Typically, this function is called to allow scrubbing of an index with a user-defined index type. If called,
              * it is the caller's responsibility to verify that the scrubbed index matches the required criteria, which are:
              * 1. For the dangling scrubber job, every index entry needs to contain the primary key of the record that
              *    generated it so that we can detect if that record is present.
              * 2. For the missing entry scrubber, the index key for the record needs to be present in the index.
-             * @param ignore True to ignore type check
              * @return this builder
              */
-            public Builder ignoreIndexTypeCheck(boolean ignore) {
-                ignoreIndexTypeCheck = ignore;
+            @ExperimentalApi
```

Comment:
The api annotations we use are `@API`, and I thought you wanted to mark this deprecated.
That probably also means you should add `@deprecated` to the javadoc. 
```suggestion
            @API(API.Status.DEPRECATED)
            @Deprecated
```